builtin/providers/aws/resource_aws_lambda_function.go

```diff
@@ -323,15 +323,26 @@ func resourceAwsLambdaFunctionRead(d *schema.ResourceData, meta interface{}) err
 	d.Set("runtime", function.Runtime)
 	d.Set("timeout", function.Timeout)
 	d.Set("kms_key_arn", function.KMSKeyArn)
-	if config := flattenLambdaVpcConfigResponse(function.VpcConfig); len(config) > 0 {
-		log.Printf("[INFO] Setting Lambda %s VPC config %#v from API", d.Id(), config)
-		err := d.Set("vpc_config", config)
-		if err != nil {
-			return fmt.Errorf("Failed setting vpc_config: %s", err)
-		}
+
+	config := flattenLambdaVpcConfigResponse(function.VpcConfig)
+	log.Printf("[INFO] Setting Lambda %s VPC config %#v from API", d.Id(), config)
+	vpcSetErr := d.Set("vpc_config", config)
+	if vpcSetErr != nil {
+		return fmt.Errorf("Failed setting vpc_config: %s", vpcSetErr)
```

Comment:
> (I would have use my own PR to reproduce, but still, thanks!)

You still can! The only way I could reproduce is if I had 2+ env vars specified in config, and then used the web UI to remove one of them. Then my next plan would show the ForceNew diff as well as the config diff